api/src/models/Config.ts

```diff
@@ -222,7 +222,20 @@ class Config {
     }
 
     get licenses(): Record<string, License> {
-        return this.ini["licenses"] || {};
+        return this.ini["licenses"] ?? {};
+    }
+
+    get agentDefaults(): Record<string, string> {
+        console.log(this.ini["agent"]["defaults"])
```

Comment:
New console.log that needs to be removed (when no longer needed).

Reply:
Made change.